lib/api/decorators/aggregation_group.rb

```diff
@@ -112,7 +112,9 @@ def set_links!(group_key)
       end
 
       def value
-        if represented == true || represented == false
+        if query.group_by_column.name == :done_ratio
+          "#{represented}%"
```

Comment:
I find it strange that we format this in the aggregation group, but don't know a better place by heart. Maybe @ulferts can chime in if we can find a better place?

Reply:
Actually we did this together and did not found a better place neither 🙈 It is quite hacky but with all other solutions the effort is not worth the benefit

Reply:
Fine with me then! :+1: